RetailItem.java

```diff
@@ -0,0 +1,28 @@
+package ItemPackage;
+
+public class RetailItem {
+    
+    //fields
+    private String description="";
+    private int unitsOnHand = 0;
+    private double price = 0.0;
+    
+    //constructor accepts values for all fields
+    public RetailItem(int unitsOnHand, double price, String description){
+        this.unitsOnHand = unitsOnHand;
+        this.price = price;
+        this.description = description;
+    }
+    
+    //Mutators
+    public void setUnitsOnHand(int unitsOnHand){this.unitsOnHand = unitsOnHand;}
+    public void setPrice(double price){this.price = price;}
+    public void setDescription(String descr){this.description = descr;}
+
+    //Accessors
```

Comment:
Very appropriate comments. Mutators and Accessors don't really need javadoc their intent is assumed by the `get` and `set` prefix :+1: